pkgs/development/python-modules/fido2/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchFromGitHub, six, cryptography }:
+
+buildPythonPackage rec {
+  pname = "fido2";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
```

Comment:
Please use `fetchPypi`: https://pypi.org/project/fido2/

Reply:
The version on pypi appears to fail the test suite.

Reply:
Then we can use `fetchFromGitHub` until https://github.com/Yubico/python-fido2/pull/8 is merged.

Reply:
Let's just leave `doCheck = false` in there instead of changing the source again.

Reply:
Then please sqash the last two commits. And it would be nice if there was a comment why `doCheck = false`.

Reply:
Done and done.